docs/dev_guide/frequently-asked-questions-faq.rst

```diff
@@ -336,5 +336,14 @@ is adjusted to avoid a bit shift that is greater than the type length:
      ...
    }
 
+**WWhy some compile commands for C/C++ files are missing in the compilation database
+generated by intercept-build when ccache is enabled?**
 
+When ccache is enabled, the object of C/C++ files created by the previous build is cached, thus some C/C++ compile commands are skipped during the execution of intercept-build.

```

Comment:
updated to: 
Also please make sure ccache (compiler cache) is disabled in your project. If ccache is enabled, intercept-build cannot generate complete compilation database, since some compile commands may be skipped as the target object(s) may be already available in cache. You can disable ccache by following command before running intercept-build command.
     .. code-block:: bash
         export CCACHE_DISABLE=1

Reply:
Okay